dev/PagerControl/PagerControl_themeresources.xaml

```diff
@@ -19,6 +19,73 @@
 
         <x:Double x:Key="PagerControlNumberPanelButtonWidth">32</x:Double>
 
+    <Style x:Key="PagerControlPageNavigationButtonStyle" TargetType="Button">
+        <Setter Property="Background" Value="Transparent"/>
+        <Setter Property="Width" Value="40"/>
+        <Setter Property="Height" Value="40"/>
+        <Setter Property="HorizontalContentAlignment" Value="Center" />
+        <Setter Property="VerticalContentAlignment" Value="Center" />
+        <Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="Button">
+                    <Grid x:Name="RootGrid" Background="{TemplateBinding Background}">
+
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="CommonStates">
+                                <VisualState x:Name="Normal" />
+
+                                <VisualState x:Name="PointerOver">
+
+                                    <Storyboard>
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="RootGrid" Storyboard.TargetProperty="Background">
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource NavigationViewButtonBackgroundPointerOver}" />
```

Comment:
Probably just forgotten when creating this style: We should replace these resources with PagerControl exclusive rsources as well - i.e. `PagerControlPageNavigationButtonBackgroundPointerOver` 

(Same as for the other similar theme resources used here.)

Reply:
That was me not paying enough attention, updated the themeresource names.